src/components/TimeControl/util.ts

```diff
@@ -433,7 +433,7 @@ export function usedForCheating(time_control) {
         case "byoyomi":
             return !(
                 time_control.main_time > QUESTIONABLE_ABSOLUTE_TIME ||
-                time_control.period_time > QUESTIONABLE_SECONDS_PER_MOVE
+                time_control.period_time * time_control.periods > QUESTIONABLE_ABSOLUTE_TIME
```

Comment:
Isn't this going to flag pretty much every live game as questionable? It's quite common to have say 5x30s byoyomi, which if i'm not mistaken, would be flagged as questionable here if the main time wasn't at least 15 minutes

Reply:
No, I think this is one of these upside down logic things.

This time setting is used for cheating if it is _not_ the case that those OR conditions are true.

So this setting is used for cheating 
```
if  (it is not the case that
      (main time is big enough or 
        there is enough time in the periods))

```

Reply:
Oh wait, you are right.

Hmm - what was I thinking!

I should have just stuck to fixing the actual bug!